src/hotspot/share/gc/shenandoah/shenandoahReferenceProcessor.hpp

```diff
@@ -26,7 +26,8 @@
 #ifndef SHARE_VM_GC_SHENANDOAH_SHENANDOAHREFERENCEPROCESSOR_HPP
 #define SHARE_VM_GC_SHENANDOAH_SHENANDOAHREFERENCEPROCESSOR_HPP
 
-#include "gc/shared/referenceDiscoverer.hpp"
+// #include "gc/shared/referenceDiscoverer.hpp"
+// #include "gc/shared/referencePolicy.hpp"
 #include "memory/allocation.hpp"
```

Comment:
Please fix that properly in an upstream change. I leave it up to you to keep the parts here, next merge should override it.

Reply:
I will figure out how to upstream that change.  Thanks.